File: Demo2019/ksyplayerlib/src/main/java/demo/playerlib/Logger.java
package demo.playerlib;

import android.util.Log;

/**
 * Created by ChenHui on 2017/4/25.
 */

class Logger {
    public static void d(String msg) {
        Log.d("Logger", msg);
    }
    public static void e(String msg) {
        Log.e("Logger", msg);
    }
}

File: Demo2020/utils/src/main/java/com/chen/utils/Logger.kt
package com.chen.utils

import android.util.Log

class Logger {


    companion object {
        val PRINT_LOG = true;
        val TAG = "LOGGER_2020";


        fun d(tag: String?, msg: String?) {
            Log.d(TAG + '_' + tag, msg)
        }

        fun d(msg: String?) {
            Log.d(TAG, msg)

        }
    }

}